### PR TITLE
Release Karaf Container 4.1.6

### PR DESCRIPTION
**Pull request by fpapon**

### CODE REVIEW

File: src/main/webapp/download.html [hunk 1]
```diff
@@ -245,19 +245,19 @@ <h3 class="text-dark">Series</h3>
...
-                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.5/apache-karaf-4.1.5.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.1.5/apache-karaf-4.1.5.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.5/apache-karaf-4.1.5.zip.sha1">SHA1</a>]
+                              <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.6/apache-karaf-4.1.6.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6.tar.gz.sha1">SHA1</a>] - 
+                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.6/apache-karaf-4.1.6.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6.zip.sha1">SHA1</a>]
```
Comment (jbonofre) [inline src/main/webapp/download.html]: For this release, we will create SHA512 as per new Apache rule.

File: src/main/webapp/download.html [hunk 2]
```diff
@@ -245,19 +245,19 @@ <h3 class="text-dark">Series</h3>
...
-                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.5/apache-karaf-4.1.5-src.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.1.5/apache-karaf-4.1.5-src.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.5/apache-karaf-4.1.5-src.zip.sha1">SHA1</a>]
+                              <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.6/apache-karaf-4.1.6-src.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6-src.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6-src.tar.gz.sha1">SHA1</a>] - 
+                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.1.6/apache-karaf-4.1.6-src.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6-src.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.1.6/apache-karaf-4.1.6-src.zip.sha1">SHA1</a>]
```
Comment (jbonofre) [inline src/main/webapp/download.html]: Same here about SHA512 instead of SHA1.

**Review by jbonofre — CHANGES_REQUESTED** [review]

**Review by jbonofre — CHANGES_REQUESTED** [review]
`archives.html` should be updated to add 4.1.5.

**Comment by fpapon** [comment]
done ;)

**Review by jbonofre — APPROVED** [review]
LGTM, thanks !

**Pull request merged**